app/code/Magento/Customer/Test/Unit/Block/Adminhtml/Edit/Tab/NewsletterTest.php

```diff
@@ -161,6 +168,55 @@ public function testInitFormCanNotShowTab()
         $this->assertSame($this->model, $this->model->initForm());
     }
 
+    /**
+     * Test getSubscriberStatusChangedDate
+     *
+     * @dataProvider getChangeStatusAtDataProvider
+     */
+    public function testGetSubscriberStatusChangedDate($statusDate, $dateExpected)
+    {
+        $customerId = 999;
+        $websiteId = 1;
+        $storeId = 1;
+        $isSubscribed = true;
+
+        $this->registryMock->method('registry')->with(RegistryConstants::CURRENT_CUSTOMER_ID)
+            ->willReturn($customerId);
+
+        $customer = $this->getMockForAbstractClass(CustomerInterface::class);
+        $customer->method('getWebsiteId')->willReturn($websiteId);
+        $customer->method('getStoreId')->willReturn($storeId);
+        $customer->method('getId')->willReturn($customerId);
+        $this->customerRepository->method('getById')->with($customerId)->willReturn($customer);
+
+        $subscriberMock = $this->getMockBuilder(Subscriber::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['loadByCustomer', 'getChangeStatusAt', 'isSubscribed', 'getData'])
+            ->getMock();
+        $this->localeDateMock->method('formatDateTime')->willReturn($statusDate);
+
+        $subscriberMock->method('loadByCustomer')->with($customerId, $websiteId)->willReturnSelf();
+        $subscriberMock->method('getChangeStatusAt')->willReturn($statusDate);
+        $subscriberMock->method('isSubscribed')->willReturn($isSubscribed);
+        $subscriberMock->method('getData')->willReturn([]);
+        $this->subscriberFactoryMock->expects($this->any())->method('create')->willReturn($subscriberMock);
+        $this->assertEquals($dateExpected, $this->model->getStatusChangedDate());
+    }
+
+    /**
+     * Data provider for testGetSubscriberStatusChangedDate
+     *
+     * @return array
+     */
+    public function getChangeStatusAtDataProvider()
+    {
+        return
+            [
+                ['',''],
+                ['Nov 22, 2023, 1:00:00 AM','Nov 22, 2023, 1:00:00 AM']
```

Comment:
To make sure that formatDate is run, it's better to make expected value different to `getChangeStatusAt`